step1-01/exercise/answers.html

```diff
@@ -0,0 +1,44 @@
+<h1>The Recipe</h1>
+<h2>4th of July Baked Beans</h2>
+
+<p>
+It's great how a single meal can take you back dozens of years. This is one of those recipes that never seems to fail to impress.
+</p>
+<p>
+I learned this recipe from the cousin of one of my college friends back in Nashville Tennessee. We had an amazing 4th of July feast which included this recipe and some bratwurst like these <a href="https://www.culinaryhill.com/wisconsin-beer-brats/" target="_blank"> Wisconsin Beer Brats</a> 
+</p>
+
+Prep Time: 10 minutes<br>
+Cook time: 3+ hours<br>
+Servings: 12<br>
+
+<h3>Ingredients:</h3>
+<ul>
+  <li>1LB Bacon chopped</li>
+ <li>3 Cans Bush's Original Baked Beans </li>
+ <li>1 Walla Wall Onion chopped </li>
+ <li>3 ground garlic cloves </li>
+ <li>4 Tablespoons of mustard </li>
+ <li>3 Tablespoons of molasses </li>
+ <li>4 Tablespoons of brown sugar </li>
+</ul>
+
+<h3>Directions:</h3>
+<ol>
+  <li>Cook bacon until it is mostly cooked, then drain most of the grease and put aside</li>
+  <li>Cook onion in remaining bacon grease</li>
+  <li>Combine onions and bacon, then add garlic, cook for a few more minutes</li>
+  <li>Add beans and get up to simmer temperature</li>
+  <li>Add mustard until your beans are nice and yellow</li>
+  <li>Add molasses until color darkens again</li>
+  <li>Add brown sugar until properly sweet</li>
+ <li>Simmer for a long time, occasionally stirring</li>
+  </ol>
+<h3>Expert Tips:</h3>
+<p>Burning off most of the liquid gives you nice, hearty, sticky beans.
+If the beans get too dry, you can always add beer!
+  </p>
+  <h3> Nutritional Information:</h3>
+Calories: lots<br>
```

Comment:
this could be a definition list
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dd

Reply:
Fixed .. Added Description List as suggested.